prisma/migrations/20220407150157_create_user_profile/migration.sql

```diff
@@ -15,6 +15,12 @@ CREATE TABLE "Profile" (
     "lastName" TEXT NOT NULL,
     "bio" TEXT,
     "githubUrl" TEXT,
+    "username" TEXT,
+    "mobile" TEXT,
+    "specialism" TEXT,
+    "startDate" TEXT,
+    "endDate" TEXT,
+    "jobTitle" TEXT,
```

Comment:
```suggestion
```

Existing migrations should not be edited, as this causes prisma to think that migrations are already applied to the db